.buildkite/pipeline.yml

```
@@ -1,13 +1,15 @@
 agents:
   queue: central
   slurm_mem: 8G
-  modules: julia/1.8.5 cuda/11.8 ucx/1.14.1_cuda-11.8 openmpi/4.1.5_cuda-11.8 hdf5/1.12.2-ompi415 nsight-systems/2022.2.1
+  modules: julia/1.9.0 cuda/11.8 ucx/1.14.1_cuda-11.8 openmpi/4.1.5_cuda-11.8 hdf5/1.12.2-ompi415 nsight-systems/2022.2.1
 
 env:
   JULIA_LOAD_PATH: "${JULIA_LOAD_PATH}:${BUILDKITE_BUILD_CHECKOUT_PATH}/.buildkite"
   OPENBLAS_NUM_THREADS: 1
   JULIA_NVTX_CALLBACKS: gc
   OMPI_MCA_opal_warn_on_missing_libcuda: 0
+  JULIA_CPU_TARGET: broadwell # arch used by GPU nodes
```

Where did you find that this env key values was needed?

---

The problem is that we may precompile on one node, and run on another, which invalidates a bunch of caches: `broadwell` are the oldest cores we have on the cluster.

See https://github.com/JuliaLang/julia/issues/50148 for some related discussion

---

Could also change it to
```suggestion
  JULIA_CPU_TARGET: 'broadwell;skylake'
```

---

Thanks, I'll try this next after retrying more recent versions

---

This also did not work, at least in that CI still failed. xref issue: https://github.com/JuliaCI/julia-buildkite/issues/298